01_introduction_to_algorithms/javascript/02_recursive_binary_search.js

```diff
@@ -0,0 +1,44 @@
+/**
```

Comment:
Please add these files to the `ES6` directory instead (or make it if it doesn't exist) since it would be good to keep ES6 and ES5 separate.